LayoutTests/crypto/subtle/ecdh-derive-bits-length-limits.html

```diff
@@ -85,7 +85,7 @@
     return Promise.resolve().then(function(result) {
         // P-256
         return Promise.all([
-            deriveBits(P256, 0, 256),
+            deriveBits(P256, 0, 0),
```

Comment:
Is this test cases for 256 bits, and similar below, not valuable anymore? How does that relate to the 0-length check?

Reply:
> Is this test cases for 256 bits, and similar below, not valuable anymore? How does that relate to the 0-length check?

The test case checked that when passing 0 as 'length', the result was the full string; now we should return an empty string instead.